shell/platform/android/io/flutter/embedding/engine/systemchannels/PlatformChannel.java

```diff
@@ -30,7 +30,7 @@
   @Nullable private PlatformMessageHandler platformMessageHandler;
 
   @NonNull @VisibleForTesting
-  protected final MethodChannel.MethodCallHandler parsingMethodCallHandler =
+  public final MethodChannel.MethodCallHandler parsingMethodCallHandler =
```

Comment:
I don't think you have to make this public.  I think you can get away with making it package visible since the tests should be in the same package.  In java that means just omitting the visibility.

Reply:
Thanks for the pointer.  The test is not in the same package, but I think it should be.  I'll split the two tests I added and create a new PlatformChannelTest.java for the one that uses this method.  Then it should work like you describe, and that organization is a lot more clear in my opinion.